Core/wbDefinitionsSF1.pas

```diff
@@ -6501,6 +6507,21 @@ procedure DefineSF1a;
     'Fifth Star Slot'
   ]);
 
+  wbPronounEnum := wbEnum([
+    {0} 'Unspecified',
+    {1} 'He/Him',
+    {2} 'She/Her',
+    {3} 'They/Them'
+  ]);
+
+  wbLRNM := wbEnum([
```

Comment:
wbLRNM would normally imply that this is a subrecord def for the LRNM subrecord.
I would call this wbLearnMethodEnum instead